css/local.css

```diff
@@ -322,70 +337,169 @@ footer p {
 
 /**** navigation ***/
 
-nav.container {
-  font-family: Noto Sans;
-  font-weight: bold;
-  margin: 0 auto;
-  padding: 1em 1em;
-  text-align: center;
+/* selected menu item */
+
+.active-menu-about a.menu-about,
+.active-menu-download a.menu-download,
+.active-menu-community a.menu-community,
+.active-menu-recipes a.menu-recipes,
+.active-menu-screenshots a.menu-screenshots,
+.active-menu-doc a.menu-doc {
```

Comment:
The common way to do this is to apply an "active" class to the element itself, rather than defining a bunch of classes that may be applied to the parent. That would make the selector much simpler and future-proof, as no specific names would be involved.